Src/Idioms/ConstructorInitializedMemberAssertion.cs

```diff
@@ -105,8 +105,7 @@ public override void Verify(ConstructorInfo constructorInfo)
             if (constructorInfo == null) throw new ArgumentNullException(nameof(constructorInfo));
 
             var parameters = constructorInfo.GetParameters();
-            if (parameters.Length == 0)
-                return;
+            if (parameters.Length == 0) return;
```

Comment:
I'd say that the previous style is better in terms of readability. Otherwise it's easy to miss that there is an eager return from the function.
The only place where I would use oneliners - assertions which are throwing exceptions.

That is though purely my opinion, so if you find this style reasonable to go - fine then ;-)